incubator/mail2most/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app:  {{ template "name" . }}
```

Comment:
please use the new labels see: https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels